app/services/builders/distribution/pull_requests/pull_request_size_repository.rb

```diff
@@ -23,7 +23,10 @@ def pr_sizes
           @pr_sizes ||= ::Events::PullRequest.where(created_at: @from..@to)
                                              .joins(:repository)
                                              .where(repositories: { name: @repository_name })
-                                             .where.not(events_pull_requests: { html_url: nil })
+                                             .where.not(
+                                               events_pull_requests: { html_url: nil },
+                                               size: nil
+                                             )
```

Comment:
can we add a test for this?

Reply:
I'd like to have tested the following cases:
- html_url = nil and size = nil
- html_url not nil and size nil
- html_url nil and size not nil
- html_url not nil and size not nil